pytest-selenium/pages/accounts.py

```diff
@@ -9,6 +9,8 @@
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.support import expected_conditions as expect
 from selenium.webdriver.support.ui import Select
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as expected
```

Comment:
These should be merged: the two EC should be `expect` or `expected` but not both and the other should be `import Select, WebDriverWait`

Reply:
If you use PyPOM's wait then you can remove `WebDriverWait`